content/rhoai/rosa-gpu/index.md

```diff
@@ -0,0 +1,380 @@
+---
+date: '2024-05-23'
+title: Running Red Hat OpenShift AI on a ROSA HCP cluster with GPUs
+tags: ["ROSA", "RHOAI", "Jupyter", "LLM", "S3"]
+authors:
+  - Diana Sari
+  - Paul Czarkowski
+---
+<!--
+## 1. Introduction
+[Large Language Models (LLMs)](https://en.wikipedia.org/wiki/Large_language_model) are a specific type of generative AI focused on processing and generating human language. They can understand, generate, and manipulate human language in response to various tasks and prompts.
+
+This guide is a simple example on how to run and deploy LLMs on a [Red Hat OpenShift Services on AWS (ROSA)](https://www.redhat.com/en/technologies/cloud-computing/openshift/aws) cluster, which is our managed service OpenShift platform on AWS, using [Red Hat OpenShift AI (RHOAI)](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai), which is formerly called Red Hat OpenShift Data Science (RHODS) and is our OpenShift platform for managing the entire lifecycle of AI/ML projects. And we will utilize [Amazon S3](https://aws.amazon.com/s3/) bucket to store the model output. In essence, here we will first install RHOAI operator and Jupyter notebook, create the S3 bucket, and then run the model.
+
+*Please note that the UI may change from time to time so what you see in the snippets below might change as well.*
+-->
+
+## Prerequisites
+
+### Tools
+
+* [AWS CLI](https://aws.amazon.com/cli/)
+* [OpenShift CLI](https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/getting-started-cli.html)
+* [ROSA CLI]()
+
+### Environment
+
+1. You will need a ROSA cluster, if you don't have one, you can follow the [ROSA guide](/experts/rosa/terraform/hcp) to create a ROSA cluster.
+
+2. You will need a GPU enabled machine pool in your ROSA cluster. If you don't have one, you can follow the [Adding GPUs to a ROSA cluster](/experts/rosa/gpu) guide to add GPUs to your cluster.
+
+## Install OpenShift Service Mesh Operator
+
+1. Deploy the Operator
+
+    ```bash
+    cat << EOF | oc apply -f -
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: servicemeshoperator
+      namespace: openshift-operators
+    spec:
+      channel: stable
+      installPlanApproval: Automatic
+      name: servicemeshoperator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+    EOF
+    ```
```

Comment:
sorry please remind me why we need to install service mesh here?

Reply:
ok nvm, i just realized RHODS actually needs both service mesh and serverless operators... btw i haven't continued testing this beyond the operator version error below so once u fixed that lmk and i'll continue then